tanstack-query-lite/react/ReactQueryDevtools.jsx

```diff
@@ -24,27 +32,7 @@ const ReactQueryDevtools = () => {
         return (
           <div key={queryHash} className="p-2">
             {JSON.stringify(queryKey, null, 2)}, {JSON.stringify({ staleTime, gcTime }, null, 2)} -{" "}
-            <span className="font-bold">
-              {(() => {
-                if (isFetching) {
-                  return <span className="text-blue-500">fetching</span>;
-                }
-
-                if (!observers.length) {
-                  return <span className="text-gray-500">inactive</span>;
-                }
-
-                if (status === "success") {
-                  return <span className="text-green-500">success</span>;
-                }
-
-                if (status === "error") {
-                  return <span className="text-red-500">error</span>;
-                }
-
-                return null;
-              })()}
-            </span>
```

Comment:
I think current the code is more intuitive, even though it uses IIFE. So, for now, I'd prefer to keep it as it is.